docs/Antora/modules/background_info/pages/streaming.adoc

```diff
@@ -1,10 +1,67 @@
 = Streaming
 
-:note-caption: TODO
-[NOTE]
-====
-* Streaming types
-* Creating and using Streaming objects
-* How to manage Streaming sources per signal
-* Streaming connection heuristic usage
-====
\ No newline at end of file
+Streaming in the openDAQ(TM) architecture refers to the client side mechanism for continuous and real-time
```

Comment:
I'd omit "client side", as it's both server and client. Maybe replace it with "transport layer mechanism".